x/mint/testutil/emissions_keeper_mocks.go

```diff
@@ -110,3 +110,15 @@ func (m *MockEmissionsKeeper) IsWhitelistAdmin(ctx context.Context, admin string
 	ret1, _ := ret[1].(error)
 	return ret0, ret1
 }
+
+func (mr *MockEmissionsKeeperMockRecorder) SetRewardCurrentBlockEmission(ctx interface{}, emission interface{}) *gomock.Call {
```

Comment:
@fernandofcampos this is one of the places where we traditionally have had to write mocks ourselves even though the files are supposed to be autogenerated